### PR TITLE
Spanish Language updated

### PR DESCRIPTION
**Pull request by TheFireRed**
Here's my Spanish translation. The missing strings have been added
to the files.

### CODE REVIEW

**Comment by kyzentun** [comment]
Add the strings added in this commit: https://github.com/stepmania/stepmania/commit/cb379505d504bdf2daf0df5db662b5ed266937d2
(Record Hold Time and the bit added to the end of EditHelpText that lists the new keys)

File: Themes/default/Languages/es.ini [hunk 1]
```diff
@@ -8,71 +11,73 @@ EventStageCounter=Fase %03i
...
+HelpText=&BACK; Salir &START; Elegir/Unirse &MENULEFT;&MENURIGHT; Mover
+EasyExplanation=Modalidad para principiantes
+HardExplanation=Solo para expertos
+OldNormalExplanation=Ni muy fácil, ni muy difícil
+NormalExplanation=¡Juega con tus canciones favoritas!
+RaveExplanation=Lucha contra un amigo o rival
+ExtendedExplanation=Varias canciones seguidas
+OniExplanation=Pon a prueba tu habilidad
+EndlessExplanation=A tope y sin parar
+NonstopExplanation=Carias canciones seguidas
```
Comment (zardoru) [inline Themes/default/Languages/es.ini]: Varias? :p

Comment (TheFireRed) [inline Themes/default/Languages/es.ini]: I'm doing a thorough proofreading with a friend and... yeah, some typos here and there, but will be fixed very, very soon, I promise! :dancer:

**Comment by kyzentun** [comment]
Post another comment in here when you are finished proof reading.

**Comment by TheFireRed** [comment]
Reread and checked! You can merge it now. Thanks!

**Pull request merged**